meilisearch-release-changelog/v0.29.0.md

```diff
@@ -0,0 +1,135 @@
+
+```bash
+curl \
+  -X POST 'http://localhost:7700/indexes/movies/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "american ninja", "matchingStrategy": "all" }'
+```
+
+Here Meilisearch will only return documents containing "american" AND "ninja".
+
+By default, Meilisearch will still return documents containing all the query words, followed by documents that don't contain all the query words.
+
+<!-- The following line should only be put in the official release changelogs, not for the RC -->
+Check out the [documentation]().
+
+Made by @ManyTheFish in #2578 and meilisearch/milli#598.
+
+## Thai language support
+
+Thai is now supported 🇹🇭
+
+If your language is missing in Meilisearch, please consider contributing to the [Meilisearch tokenizer](https://github.com/meilisearch/charabia) by following the CONTRIBUTING.md file and integrating your intended normalizer/segmenter.
+
+The whole core team is available to help you contribute. Feel free to ask for any clarifications!
+
+Made by our dear contributor @aFluffyHotdog and by @ManyTheFish in meilisearch/charabia#114, meilisearch/charabia#115, meilisearch/milli##611, and #2636.
+
+## Performance improvements for indexing
+
+We improved the indexing speed, especially for `filterableAttributes` and document update and deletion.
+
+For people wanting information about the main improvements, check out the following issues: meilisearch/product#513 and meilisearch/product#520.
+
+Made by @loiclec and @irevoire in #2689, #2690, meilisearch/milli#607, meilisearch/milli#606, meilisearch/milli#587, meilisearch/milli#557,meilisearch/milli#579, and meilisearch/milli#590.
+
+## Filtering: introduce the `EXISTS` and `IN` operators
+
+You can now use the `EXISTS` and `IN` operators with the `filter` search parameter during [search](https://docs.meilisearch.com/reference/api/search.html#search-in-an-index-with-post-route).
+
+With the following set of documents:
+
+```json
+{ "id": 1, "product": "T-shirt", "price": 20, "color": "yellow" }
+{ "id": 2, "product": "T-shirt",              "color": "red" }
+{ "id": 3, "product": "T-shirt", "price": 30 }
+{ "id": 4, "product": "T-shirt", "price": 40 }
+```
+
+- The filter `price EXISTS` will select documents `1`, `3`, and `4`.
+- The filter `price IN [30, 20]` will select documents `1` and `3`.
+
+These operators can be combined with the already existing `NOT` (`!=`) operator.
+More about [filtering](https://docs.meilisearch.com/learn/getting_started/filtering_and_sorting.html).
+
+Made by @loiclec in #2636, meilisearch/milli#596, and meilisearch/milli#556.
+
+## Key management
+
+When managing [keys](https://docs.meilisearch.com/learn/security/master_api_keys.html#protecting-a-meilisearch-instance), the `*` helper can now be used to express a set of actions in the `actions` array.
+
+For example, instead of specifying `indexes.create`, `indexes.get`, `indexes.update`, `indexes.delete`; it's now possible to specify `indexes.*`.
```

Comment:
```suggestion
For example, instead of specifying `indexes.create`, `indexes.get`, `indexes.update`, `indexes.delete`; you can now use `indexes.*`.
```